install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -428,10 +428,10 @@ project that has a claim to them.
 - Configure xref:nfs-reclaiming-resources[reclamation of discarded resources] for
 each PV.
 
-They are many ways that you can use scripts to automate the above tasks. You can
+There are many ways that you can use scripts to automate the preceding tasks. You can
 use an
-link:https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_storage_nfs[example
-Ansible playbook] to help you get started.
+link:https://github.com/openshift/openshift-ansible/tree/release-3.11/roles/openshift_storage_nfs[example
+Ansible playbook] that is associated with the {product-title} 3.11 release to help you get started.
```

Comment:
@mikemckiernan can you please replace "3.11" with `{product-version}`? Sorry, I should have spotted this before.

Reply:
I think this is ok since this is only being merged to 3.11.

Reply:
Hi @pneedle-rh, my goal was to indicate that the URL goes to a 3.11 release playbook, so I'd never want that "3.11" to change unless I _also_ changed the link from 3.11 to something else.

Reply:
Ok thank you Mike for clarifying that. I think in this case it would be ok either way, because we are only merging directly to the 3.11 release branch.